test/functional/test_runner.py

```diff
@@ -225,6 +225,7 @@
     'feature_loadblock.py',
     'p2p_dos_header_tree.py',
     'p2p_unrequested_blocks.py',
+    'p2p_cfilters.py',
```

Comment:
After the change to `-peerblockfilters`, should the test be renamed as well? e.g. `p2p_peerblockfilters.py`

Reply:
done